test-fixtures/src/bin/dictionary-lookup.rs

```diff
@@ -10,10 +10,18 @@ fn main() {
             "animals".len(),
             &mut dict_handle as *mut _,
         );
-        assert_eq!(res, FastlyStatus::OK, "Failed to open dictionary");
+        assert_eq!(res, FastlyStatus::OK, "Failed to open config-store");
```

Comment:
Maybe remove this file?

Reply:
I'll change it back to `dictionary`, thanks!